packages/components/app/styles/components/tabs.scss

```diff
@@ -8,30 +8,32 @@ $indicator-height: 3px;
 
 // Sub-components:
 .hds-tabs__tablist-wrapper {
-  padding-bottom: round(calc($indicator-height / 2));
-  // overflow-x: auto;
-  // -webkit-overflow-scrolling: touch;
-  // white-space: nowrap; // move
-}
-
-.hds-tabs__tablist {
   position: relative;
-  display: flex;
-  margin: 0;
-  padding: 0;
+  padding-bottom: round(calc($indicator-height / 2));
```

Comment:
With this change the overall height of the `Tabs` container becomes `38px` (`36px` a tab + `2px` the border bottom), while the height of the `Tabs` (the whole component) in Figma is `36px` as well as a single `Tab`, which is also `36px` in Figma.